### DISCUSSION TITLE
Main.css version not updating

### DISCUSSION THREAD

**Discussion by jamessilverstripe** [bootscore/bootscore#876] [Q&A]
I've recently started using the latest version of Bootscore and have encountered an issue I've not had with previous versions. The main.css file is compiling and updating correctly but the version number in the head is set to "?ver=6.0.0". If I change the ver number or remove it completely, the correct styles load. I have turned off my cache, tried deleting main.css, have tried dev mode in wp config but nothing is working. Please help 🙏

**Comment by crftwrk** [comment]
See https://github.com/bootscore/bootscore/issues/877
